src/pilot/install-director.sh

```diff
@@ -361,14 +361,6 @@ for container in "ironic_pxe_tftp"  "ironic_conductor" "ironic_api" ;
     run_on_container "${container}" "rm -f /usr/lib/python3.6/site-packages/dracclient/wsman.pyo"
     echo "## Done"
 
-    echo
-    echo "## Patching Ironic iDRAC driver idrac_card.py on ${container}..."
-    upload_file_to_container "${container}" "${HOME}/pilot/idrac_card_groupid.patch" "/tmp/idrac_card_groupid.patch"
-    run_on_container "${container}" "patch -b -s /usr/lib/python3.6/site-packages/dracclient/resources/idrac_card.py /tmp/idrac_card_groupid.patch"
-    run_on_container "${container}" "rm -f /usr/lib/python3.6/site-packages/dracclient/resources/idrac_card.pyc"
-    run_on_container "${container}" "rm -f /usr/lib/python3.6/site-packages/dracclient/resources/idrac_card.pyo"
-    echo "## Done"
-
```

Comment:
For consistency and sanity, I suggest having it be consistent across all installations, at least on the undercloud.

Reply:
The versions of the module(s) installed on the undercloud base OS ( and used by config idrac ) are different than the ones running on the undercloud containers. If we wanted to patch both we d need x2 set of patches (double the headache for each zStream update).
In this particular case, since it's only affecting a config idrac use case,  i think patching on the os only is fine.